src/board/__tests__/board.test.tsx

```diff
@@ -99,4 +106,62 @@ describe("Board", () => {
     itemDragHandle.focus();
     itemDragHandle.keydown(KeyCode.escape);
   });
+
+  test("applies reorder operation classname", async () => {
+    const { container } = render(
+      <Board
+        items={[
+          { id: "1", data: { title: "Item 1" } },
+          { id: "2", data: { title: "Item 2" } },
+        ]}
+        renderItem={(item) => <BoardItem i18nStrings={itemI18nStrings}>{item.data.title}</BoardItem>}
+        onItemsChange={() => undefined}
+        i18nStrings={i18nStrings}
+        empty="No items"
+      />
+    );
+
+    const reorderClass = boardStyles["current-operation-reorder"];
+    expect(container.ownerDocument.body).not.toHaveClass(reorderClass);
+
+    const handle = createWrapper().findBoardItem()!.findDragHandle()!.getElement();
+    const user = userEvent.setup({ pointerEventsCheck: PointerEventsCheckLevel.Never });
+
+    // Start a reorder operation and check that the class is set
+    await user.pointer({ keys: "[MouseLeft>]", target: handle });
```

Comment:
Why did you need this library here? What did not work when dispatching mousedown directly?

If we have a plan of converting this code into test-util, it will be more difficult when using an external dependency

Reply:
There were a couple of problems:

1. I couldn't create a proper `pointerdown` event with `fireEvent.pointerDown` because it doesn't let me specify the `button` event parameter, see https://github.com/testing-library/react-testing-library/issues/838.
2. I couldn't (easily) create a custom `PointerEvent` object because this is not supported in jsdom, see https://github.com/jsdom/jsdom/issues/2527.

Reply:
https://github.com/jsdom/jsdom/issues/2527 also mentions a workaround – using MouseEvent constructor instead

I tried this and it worked, see #236 

user-event library makes it work the same way, but more obscure, because it does a lot more additional side effects

Reply:
This is also a strong signal that we need an official test-util for this scenario (because releasing cursor on `window` is implementation detail).

Can we use #236 for now instead of an extra dependency and I will later open a PR to introduce the official util?

Reply:
Thanks! I swear I tried `MouseEvent("pointerdown")` and it didn't work for me... I must have done something else wrong. Will update this PR.